### DISCUSSION TITLE
no matches found: spacy[apple]

### DISCUSSION THREAD

**Discussion by geg00** [explosion/spaCy#10014] [Help: Installation] [answered]
## How to reproduce the behaviour
pip install -U spacy[apple]

## Your Environment

* Operating System:  
*  
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro18,2
  Chip:	Apple M1 Max
  Total Number of Cores:	10 (8 performance and 2 efficiency)
  Memory:	32 GB
  System Firmware Version:	7429.61.2
  OS Loader Version:	7429.61.2
  Serial Number (system):	FVQWJWV42G
  Hardware UUID:	E5D06237-B789-574E-9187-8EEBED74CE1A
  Provisioning UDID:	00006001-001C71442EF8801E
  Activation Lock Status:	Enabled

* Python Version Used:
python                    
Python 3.9.7 (default, Sep 16 2021, 08:50:36) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 

spaCy Version Used:
I'm trying to install the latest 

* Environment Information:

==> I'm getting the following error when I try to install the spacy[apple]

python -m venv .env
source .env/bin/activate
pip install -U pip setuptools wheel
pip install -U spacy[apple]
python -m spacy download en_core_web_sm

pip install -U spacy[apple]
zsh: no matches found: spacy[apple]

**Comment by adrianeboyd** [comment, marked as answer]
It should work to put quotes around the package+extras specification (so that your shell zsh doesn't try to interpret it further):

```bash
pip install -U "spacy[apple]"
```

Most shells like bash don't this with square brackets, so we haven't added quotes in the default commands. It might make sense to modify this for the `[apple]` instructions, since it's the default for OS X.

**Comment by geg00** [comment]
It may be a good idea to update the documentation on the page
https://spacy.io/usage
